packages/attribute-registry-consumer/.env

```diff
@@ -6,6 +6,7 @@ KAFKA_CLIENT_ID="my-app"
 KAFKA_GROUP_ID="my-group"
 KAFKA_BROKERS="localhost:9092"
 KAFKA_DISABLE_AWS_IAM_AUTH="true"
+KAFKA_TOPICS="event-store.attribute.events"
```

Comment:
This was missing, but present in all other readmodel writer services. Added it so that I was able to do a smoke test of some API calls to check that the readmodel writes still work.